README.md

```diff
@@ -2,47 +2,64 @@
 
 [![Codacy Badge](https://app.codacy.com/project/badge/Grade/db5eda4c50b24c008b2c19f1117c9cef)](https://www.codacy.com/gh/codacy/codacy-coverage-reporter-action/dashboard?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=codacy/codacy-coverage-reporter-action&amp;utm_campaign=Badge_Grade)
 
-> 🤖 Automated multi-language coverage reporter for [Codacy](https://www.codacy.com/).  
-> For more information about Codacy see our [5 minutes quickstart](https://docs.codacy.com/getting-started/codacy-quickstart/).
+GitHub Action for [uploading coverage reports to Codacy](https://docs.codacy.com/coverage-reporter/#uploading-coverage) on all commits and pull requests, tracking code coverage, and returning code coverage issues.
 
-This action runs the [codacy-coverage-reporter](https://github.com/codacy/codacy-coverage-reporter) for all commits and pull requests with your Codacy configuration.
+[Codacy](https://www.codacy.com/) is an automated code review tool that makes it easy to ensure your team is writing high-quality code by analyzing more than 40 programming languages such as PHP, JavaScript, Python, Java, and Ruby. Codacy allows you to define your own quality rules, code patterns and quality settings you'd like to enforce to prevent issues on your codebase.
 
-## Usage
+<br/>
 
-Create a new workflow `.yml` file in the `.github/workflows/` directory.
+<a href="https://www.codacy.com" target="_blank"><img src="images/codacy-logo.svg" alt="Codacy" width="400"/></a>
 
-### .github/workflows/codacy-coverage-reporter.yml
+<br/>
 
-```yml
-name: codacy-coverage-reporter
+## Coverage using your project’s quality settings
 
-on: ["push"]
+Before setting up Codacy Coverage Reporter GitHub Action you must have tests and use tools to [generate coverage reports](https://docs.codacy.com/coverage-reporter/#generating-coverage) for the source code files in your repository.
 
-jobs:
-  codacy-coverage-reporter:
-    runs-on: ubuntu-latest
-    name: codacy-coverage-reporter
-    steps:
-      - uses: actions/checkout@v2
-      - name: Run codacy-coverage-reporter
-        uses: codacy/codacy-coverage-reporter-action@v1
-        with:
-          project-token: ${{ secrets.CODACY_PROJECT_TOKEN }}
-          # or
-          # api-token: ${{ secrets.CODACY_API_TOKEN }}
-          coverage-reports: cobertura.xml
-```
+The Codacy Coverage Reporter GitHub Action:
 
-> Get the latest version from the [Releases page](https://github.com/codacy/codacy-coverage-reporter-action/releases).
+-   Uploads coverage report data to Codacy.
+-   Prints the coverage report upload status on the console, which is visible on the workflow panel of the GitHub Action.
+-   Fails the workflow if code coverage falls below your project’s [quality settings](https://docs.codacy.com/repositories-configure/adjusting-quality-settings/#gates).
 
-## Workflow options
+To use the GitHub Action with default settings:
```

Comment:
By removing the list items above, we're left with just one item ("Uploads coverage report data to Codacy") so it doesn't make sense to have a list anymore. But we can still make it explicit that the use case is for uploading coverage to Codacy in the sentence that introduces the procedure:

```suggestion
To upload coverage to Codacy using the GitHub Action with default settings:
```